src/bornhack/settings.py

```diff
@@ -69,10 +70,10 @@ def local_dir(entry):
 # USE_I18N = True
 # USE_L10N = True
 USE_TZ = True
-SHORT_DATE_FORMAT = "d/m-Y"
-DATE_FORMAT = "d/m-Y"
-DATETIME_FORMAT = "d/m-Y H:i"
-TIME_FORMAT = "H:i"
+SHORT_DATE_FORMAT = "Ymd"
+DATE_FORMAT = "l, M jS, Y"
+DATETIME_FORMAT = "l, M jS, Y, H:i (e)"
```

Comment:
What is `e`?

Reply:
timezone abbreviation, CET right now, CEST during bornhack